app/javascript/packages/document-capture/components/acuant-capture.tsx

```diff
@@ -129,19 +155,16 @@ function getDocumentTypeLabel(documentType) {
   }
 }
 
-/**
- * @param {import('./acuant-camera').AcuantCaptureFailureError} error
- * @param {string=} code
- *
- * @return {string}
- */
-export function getNormalizedAcuantCaptureFailureMessage(error, code) {
+export function getNormalizedAcuantCaptureFailureMessage(
+  error: AcuantCaptureFailureError,
+  code: string | undefined,
+): string {
   if (isAcuantCameraAccessFailure(error)) {
     return 'User or system denied camera access';
   }
 
-  const { REPEAT_FAIL_CODE, SEQUENCE_BREAK_CODE } = /** @type {AcuantGlobal} */ (window)
-    .AcuantJavascriptWebSdk;
+  const { REPEAT_FAIL_CODE, SEQUENCE_BREAK_CODE } =
+    /** @type {AcuantGlobal} */ window.AcuantJavascriptWebSdk;
```

Comment:
if this is TSX now it should be casted with `as` right?
```suggestion
    (window as AcuantGlobal).AcuantJavascriptWebSdk;
```